api_tests/users/views/test_user_detail.py

```diff
@@ -842,8 +842,434 @@ def test_partial_put_user_logged_in(self, app, user_one, url_user_one):
         assert user_one.suffix == 'The Millionth'
         assert user_one.social['github'] == 'even_newer_github'
 
-    def test_put_user_logged_in(
-            self, app, user_one, data_new_user_one, url_user_one):
+    def test_user_put_schools_200(self, app, user_one, url_user_one):
+        # Tests to make sure institution is not empty string
+        res = app.put_json_api(url_user_one, {
+            'data': {
+                'id': user_one._id,
+                'type': 'users',
+                'attributes': {
+                    'full_name': user_one.fullname,
+                    'education': [{'degree': '',
+                                   'startYear': 1991,
+                                   'startMonth': 9,
+                                   'endYear': 1992,
+                                   'endMonth': 9,
+                                   'ongoing': False,
+                                   'department': '',
+                                   'institution': 'Fake U'
+                                   }]
+                },
+            }
+        }, auth=user_one.auth)
+        user_one.reload()
+        assert res.status_code == 200
+        assert user_one.schools == [{'degree': '',
+                                   'startYear': 1991,
+                                   'startMonth': 9,
+                                   'endYear': 1992,
+                                   'endMonth': 9,
+                                   'ongoing': False,
+                                   'department': '',
+                                   'institution': 'Fake U'
+                                     }]
+
+    def test_user_put_schools_401(self, app, user_one, url_user_one):
+        # Tests to make sure institution is not empty string
+        res = app.put_json_api(url_user_one, {
+            'data': {
+                'id': user_one._id,
+                'type': 'users',
+                'attributes': {
+                    'full_name': user_one.fullname,
+                    'education': [{'degree': '',
+                                   'startYear': 1991,
+                                   'startMonth': 9,
+                                   'endYear': 1992,
+                                   'endMonth': 9,
+                                   'ongoing': False,
+                                   'department': '',
+                                   'institution': 'Fake U'
+                                   }]
+                },
+            }
+        }, expect_errors=True)
+        assert res.status_code == 401
```

Comment:
Can you also add a test_user_put_schools_403/test_user_put_jobs_403 test for users trying to edit someone else's profile info?